### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by glc-mvion**
Here is my modest contribution to the QMK software, a french translation of the website ! :tada: 

I felt free to manipulate a bit some sentences to be more "french friendly" more than translate word by word.

Also it has a great chance that typos invited to the party, even if I re-re-read my text :eyes:

### CODE REVIEW

**Comment by Darkpingouin** [comment]
Hey, I am reviewing your files and there are some typos, not too much so I won't open a pull request for them as it would not be usefull.
- File ` _i18n/fr/coc.md ` line **14** you typed `régir` instead of `réagir`
- File `_i18n/fr/sidebar.md` line **1** you type `n'import` instead of  `n'importe`

**Comment by Darkpingouin** [comment]
Fixes https://github.com/qmk/qmk_firmware/issues/6924
I would also Add the labels hactkoberfest and translation to your PR

**Comment by noroadsleft** [comment]
@zekth Would you mind reviewing this translation please?

File: _i18n/fr/coc.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: "Code de conduite de QMK"
+layout: qmk
+permalink: "/fr/coc/"
+lang: fr
+---
+
+# Code de conduite
+
+QMK s'efforce d'être une communauté, accueillante, tolérante et inclusive. Nous encourageons toutes les contributions indépendamment de l'âge, de quelconque handicap, de l'appartenance éthnique, du genre, du niveau d'expérience technique, de la nationalité, de l'apparence physique, des orientations politiques, de la race, de la religion ou de l'orientation et de l'identification sexuelle.
```
Comment (zekth) [inline _i18n/fr/coc.md:10]:
```suggestion
QMK s'éfforce d'être une communauté, accueillante, tolérante et inclusive. Nous encourageons toutes les contributions indépendamment de l'âge, de quelconque handicap, de l'appartenance ethnique, du genre, du niveau d'expérience technique, de la nationalité, de l'apparence physique, des orientations politiques, de la race, de la religion ou de l'orientation et de l'identification sexuelle.
```

File: _i18n/fr/coc.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+> "Une réponse douce calme la fureur, Mais une parole dure excite la colère."
+
+Nous attendons de nos utilisateurs, contributeurs et collaborateurs qu'ils se traitent avec gentillesse et respect, qu'ils aient de bonnes intentions, de corriger avec pédagogie, quand cela est possible, plutôt que de régir de façon disproportionnée. Même si notre but est d'être aussi précis que possible, la compréhension et la gentillesse sont plus précieuses que l'exactitude.
```
Comment (zekth) [inline _i18n/fr/coc.md:14]:
```suggestion
Nous attendons de nos utilisateurs, contributeurs et collaborateurs qu'ils se traitent avec gentillesse et respect, qu'ils aient de bonnes intentions, de corriger avec pédagogie, quand cela est possible, plutôt que de réagir de façon disproportionnée. Même si notre but est d'être aussi précis que possible, la compréhension et la gentillesse sont plus précieuses que l'exactitude.
```

File: _i18n/fr/sidebar.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+Le but du projet QMK est de développer un firmware complètement personnalisable, puissant et facile à utiliser pour tous type de projet de clavier ou autre, et de fournir une aide bienveillante, encourageante et utile ainsi que des retours pour des personnes avec n'import quelle expérience en développement logiciel.
```
Comment (zekth) [inline _i18n/fr/sidebar.md:1]:
```suggestion
Le but du projet QMK est de développer un firmware complètement personnalisable, puissant et facile à utiliser pour tous type de projet de clavier ou autre, et de fournir une aide bienveillante, encourageante et utile ainsi que des retours pour des personnes avec n'importe quelle expérience en développement logiciel.
```

**Review by zekth — CHANGES_REQUESTED** [review]
Minor fix. Otherwise ok to me

File: _i18n/fr/coc.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+* l'intimidation délibérée, le "stalking"
+* les insultes, commentaires désobligeants, les attaques personnelles ou politiques
+* poster des informations personnelles sans l'autorisation explicite de leur propriétaire
+* perturber de quelque façon la discussion d'autrui
+* n'importe quel comportement qui pourrait être considéré comme inapproprié dans un contexte professionnel
+* prôner ou encourager n'importe lequel des comportement ci-dessus
+
+# Signaler
+
+Si quelqu'un viole ce Code de Conduite, veuillez envoyer un email à hello arobase qmk.fm ou contactez l'un des collaborateur. Toutes les plaintes seront évaluées et prises en considération.
```
Comment (Ph0tonic) [inline _i18n/fr/coc.md:29]: Si quelqu'un viole ce Code de Conduite, veuillez envoyer un email à hello@qmk.fm ou contactez l'un des collaborateur. Toutes les plaintes seront évaluées et prises en considération.

Comment (Ph0tonic) [reply _i18n/fr/coc.md:29]: Missing a real arobase.

**Review by Ph0tonic — CHANGES_REQUESTED** [review]
Small details

File: _i18n/fr/support.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+* [Produits dérivés QMK sur olkb.com](https://olkb.com/parts)
+* [Boutique Teespring](https://teespring.com/stores/qmk)
+* [Faites un don via Donorbox](https://donorbox.org/qmk)
+* [Contribuez !](https://github.com/qmk/qmk_firmware/issues)
+
+## Comment nous utilisons vos donations
+
+* Couvrir les frais des serveurs et d'infrastructure
+* Couvrir les frais de prototypages pour des produits QMK comme le [Proton C](/proton-c/)
+* Envoyer des goodies à des meetups pour offrir ou pour des concours. Envoyez nous un email si vous êtes intéressés ! hello arobase qmk.fm
```
Comment (Ph0tonic) [inline _i18n/fr/support.md:19]: Missing a real arobase:
`* Envoyer des goodies à des meetups pour offrir ou pour des concours. Envoyez nous un email si vous êtes intéressés ! hello@qmk.fm`

File: _i18n/fr/coc.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
```
Comment (noroadsleft) [inline _i18n/fr/coc.md:29]:
```suggestion
Si quelqu'un viole ce Code de Conduite, veuillez envoyer un email à hello@qmk.fm ou contactez l'un des collaborateur. Toutes les plaintes seront évaluées et prises en considération.
```

File: _i18n/fr/support.md (hunk 5 above)
Comment (noroadsleft) [inline _i18n/fr/support.md:19]:
```suggestion
* Envoyer des goodies à des meetups pour offrir ou pour des concours. Envoyez nous un email si vous êtes intéressés ! hello@qmk.fm
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
Changes as requested by @Ph0tonic

**Review by Ph0tonic — APPROVED** [review]

**Comment by noroadsleft** [comment]
Going to merge this as-is and then do another PR that integrates suggestions from @Darkpingouin, @zekth and @Ph0tonic.

I don't like writing to other user's branches without their explicit permission, and I'd like to get this across the line.

Thank you all for your contributions!

**Review by noroadsleft — APPROVED** [review]

**Pull request merged**